AV-45749: Update sample queries (#35)
couchbaselabs/docs-devex: modules/n1ql/pages/n1ql-intro/queriesandresults.adoc

When the query context is set, you can refer to a collection using just the collection name, without having to enter the keyspace path.
When the query context is not set, it defaults to the `default` namespace, with no bucket, scope, or collection.

To set the query context in the Query Workbench:

> . Go to *Data Tools* > *Query*.

Reviewer: ```suggestion
. Go to menu:Data Tools[Query].
```

> . In the *Data Insights* sidebar, expand the database.

Reviewer: ```suggestion
. In the *Data Insights* area, expand the database.
```

> . Click the scope where you want to run your query.

Reviewer 2: This should be a reference to the query context section in the Query Workbench page. The Query Workbench page should be updated to contain the correct instructions for setting the query context in Elixir.

Author: That sounds like a separate PR. That page is in cloud, not devex.

* Named placeholders take the form of `$name`.
This is particularly useful when there are many query parameters, and ensuring that they are all in the correct order may be cumbersome.

> You can set query parameter values when you run the query, the Query Workbench, or the {sqlpp} REST API.

Reviewer: ```suggestion
You can set query parameter values when you run the query, use the Query Workbench, or use the {sqlpp} REST API.
```

Author: Good catch :)

* Named placeholders take the form of `$name`.
This is particularly useful when there are many query parameters, and ensuring that they are all in the correct order may be cumbersome.

> You can set query parameter values when you run the query, use the Query Workbench, or use the {sqlpp} REST API.

Reviewer 2: I'd rather comment out the reference to cbq (or put it behind a feature flag) because ultimately, it needs to be replaced with a reference to the cb shell.

```suggestion
You can set query parameter values when you run the query, using
// the cbq query shell,
the Query Workbench, or the {sqlpp} REST API.
```
